tests/agent/build_vllm_gaudi.sh

```diff
@@ -5,13 +5,8 @@ function build_vllm_docker_images() {
     echo "Building the vllm docker images"
     cd $WORKDIR
     echo $WORKPATH
-    if [ ! -d "./vllm" ]; then
-        git clone https://github.com/HabanaAI/vllm-fork.git
-    fi
-    cd ./vllm-fork
-    VLLM_VER=v0.6.6.post1+Gaudi-1.20.0
-    echo "Check out vLLM tag ${VLLM_VER}"
-    git checkout ${VLLM_VER} &> /dev/null
+    VLLM_FORK_VER=v0.6.6.post1+Gaudi-1.20.0
```

Comment:
Why these and the other scripts are not including this variable from the shell script added for defining them?